modules/project-docs/pages/compatibility.adoc

```diff
@@ -58,9 +58,12 @@ It is best to upgrade either the SDK or the Couchbase version you are using.
 | *✖*
 | *✖*
 | *◎*
-| *✔*
+| *◎* _(see below)_
 |===
 
+IMPORTANT: when running against Server 7.0β, Java SDK *3.1.6* is the latest compatible version.
+Later SDK 3.1.x releases will only be able to use the Default Collection, due to a change of API.
+
```

Comment:
I think we should outline the change and future plan first (for reasons discussed), then discuss the versions for beta users. 
How about:
"IMPORTANT: An API change in the planned release version of Couchbase Server 7.0 means changes in the latest versions of the SDK which render these new SDK versions incompatible with the _beta_ versions of 7.0.
Java SDK *3.1.6* is the latest version compatible with Server 7.0β.
Later SDK 3.1.x releases will only be fully compatible with the full release version of 7.0, except in the case of just running against _default_ Scopes and Collections with the beta 7.0 Server."
Could make this a partial (once settled on) and pull it in to each compat doc.

Reply:
Sounds good! Bringing in @AV25242 to give his opinion on this and rest of PR.

Reply:
> Sounds good! Bringing in @AV25242 to give his opinion on this and rest of PR.

I had mentioned it above but 3.1.6 for instance is not the one thats compatible with 7.0 Beta refer to the currently compatibility matrix page 
   https://www.couchbase.com/products/developer-sdk/compatibility
As an example for Java SDK versions ranging from SDK 3.0.9 through 3.1.3 will be compatible

Reply:
Thanks - is that document correct for all cases?

We were looking at the developer-updated GDoc and taking the complement (e.g. assuming that what works for Beta does NOT work for GA and v.v.)

Does this mean that 3.1.4-3.1.5 don't work correctly with either 7.0beta *or* 7.0?